spec/rubocop/cop/lint/debugger_spec.rb

```diff
@@ -50,6 +50,51 @@
                 save_screenshot))
   end
 
+  it 'reports an offense for a debugger call with an argument' do
```

Comment:
There's a bit of duplication in the specs for this cop. Maybe you can introduce some shared examples to reduce it.